contracts/modules/Exec.sol

```diff
@@ -32,6 +32,8 @@ contract Exec is BaseLogic {
         bytes result;
     }
 
+    error BatchDispatchSimulation(EulerBatchItemResponse[] simulation);
```

Comment:
Do we need to add this to the exported interface? If so, we'll want natspec docs and might need a tweak to the perl script.

Reply:
yea, we need that :+1: